src/test/ui/lint/lint-temporary-cstring-as-ptr.stderr

```diff
@@ -1,16 +1,16 @@
-error: you are getting the inner pointer of a temporary `CString`
-  --> $DIR/lint-temporary-cstring-as-ptr.rs:7:13
+error: getting the inner pointer of a temporary `CString`
+  --> $DIR/lint-temporary-cstring-as-ptr.rs:6:13
    |
 LL |     let s = CString::new("some text").unwrap().as_ptr();
-   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^ this pointer will be invalid
```

Comment:
Oh, I thought that the span would point to `as_ptr()`...

Reply:
Whoops. Yup that definitely makes more sense.